website/docs/r/datazone_glossary.html.markdown

```diff
@@ -0,0 +1,133 @@
+---
+subcategory: "DataZone"
+layout: "aws"
+page_title: "AWS: aws_datazone_glossary"
+description: |-
+  Terraform resource for managing an AWS DataZone Glossary.
+---
+# Resource: aws_datazone_glossary
+
+Terraform resource for managing an AWS DataZone Glossary.
+
+## Example Usage
+
+```terraform
+
+resource "aws_iam_role" "domain_execution_role" {
+  name = "example_name"
+  assume_role_policy = jsonencode({
+    Version = "2012-10-17"
+    Statement = [
+      {
+        Action = ["sts:AssumeRole", "sts:TagSession"]
+        Effect = "Allow"
+        Principal = {
+          Service = "datazone.amazonaws.com"
+        }
+      },
+      {
+        Action = ["sts:AssumeRole", "sts:TagSession"]
+        Effect = "Allow"
+        Principal = {
+          Service = "cloudformation.amazonaws.com"
+        }
+      },
+    ]
+  })
+
+  inline_policy {
+    name = "example_name"
+    policy = jsonencode({
+      Version = "2012-10-17"
+      Statement = [
+        {
+          Action = [
+            "datazone:*",
+            "ram:*",
+            "sso:*",
+            "kms:*",
+          ]
+          Effect   = "Allow"
+          Resource = "*"
+        },
+      ]
+    })
+  }
+}
+
+resource "aws_datazone_domain" "test" {
+  name                  = "example_name"
+  domain_execution_role = aws_iam_role.domain_execution_role.arn
+}
+
+resource "aws_security_group" "test" {
+  name = "example_name"
+}
+
+resource "aws_datazone_project" "test" {
+  domain_identifier   = aws_datazone_domain.test.id
+  glossary_terms      = ["2N8w6XJCwZf"]
+  name                = "example_name"
+  description         = "desc"
+  skip_deletion_check = true
+}
+
+
+resource "aws_datazone_glossary" "test" {
+  client_token              = "example_token"
```

Comment:
`client_token` should be set on the request by the provider. It should not be mentioned as an input value